### DISCUSSION TITLE
Table run_status

### DISCUSSION THREAD

**Discussion by petr-klinger** [cybertec-postgresql/pg_timetable#384] [Q&A] [answered]
I uploaded a new version and found that the run_status table was completely deleted. Unable to find information about each task_id at this time.

**Comment by pashagolub** [comment, marked as answer]
Hey Petr,

thanks for your request. We found this information is doubled. You may find detailed tasks information in `timetable.execution_log`, e.g.
```sql
-- find all tasks for chain `#4` executed by the `loader` client where error happened
SELECT * FROM timetable.execution_log WHERE chain_id = 4 AND client_name = 'loader' AND returncode != 0;
```
execution_log
---
| chain_id | task_id | last_run | finished | pid | returncode | kind | command | output | client_name | 
| ---: | ---: | --- | --- | ---: | ---: | --- | --- | --- | --- | 
| 4 | 6 | 2022-01-28 17:44:06.027329+01 | 2022-01-28 17:44:09.054895+01 | 23580 | -1 | SQL | SELECT sleepy_func($1) | timeout: context canceled | loader |
